runners/samza/src/main/java/org/apache/beam/runners/samza/runtime/Op.java

```diff
@@ -61,4 +61,8 @@ default void processSideInputWatermark(Instant watermark, OpEmitter<OutT> emitte
   default void processTimer(KeyedTimerData<K> keyedTimerData, OpEmitter<OutT> emitter) {}
 
   default void close() {}
+
+  default String getFullOpName() {
```

Comment:
`getName` or `getFullName`

Op is redundant.

Reply:
fixed